DuggaSys/tests/showDuggaservice_test.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+include "../../Shared/test.php";
+
+$testsData = array(
+    'Get active users' => array(
+        'expected-output' => '{"debug":"[Guest] Missing hash\/password\/variant!","param":"{}","answer":"UNK","danswer":"UNK","score":0,"highscoremode":"","grade":"UNK","submitted":"","marked":"","deadline":"UNK","release":"UNK","files":[],"userfeedback":"UNK","feedbackquestion":"UNK","variant":"UNK","ishashindb":false,"variantsize":"UNK","variantvalue":"UNK","password":"UNK","hashvariant":"UNK","isFileSubmitted":"UNK","isTeacher":0,"variants":[],"duggaTitle":"UNK","hash":"UNK","hashpwd":"UNK","opt":"UNK","link":"UNK"}',
+        'service' => 'localhost/LenaSYS/DuggaSys/showDuggaservice.php',
```

Comment:
ALL 'service' addresses are wrong, It should be:
https://cms.webug.se/root/G2/students/"YOUR USER"/LenaSYS/DuggaSys/showDuggaservice.php